CHANGELOG.md

```diff
@@ -74,7 +53,48 @@ can help deploy the Helm chart in a cluster with security requirements.
   `hub-secret`, which will affect anyone who use the `hub.existingSecret`
   option.
 
+#### Release highlights
+
+- An ability to configure environment variables in pods with a k8s native syntax
+  has been added. This allows you to reference and mount a field in a k8s Secret
+  as an environment variable for example. For more information, read [about
+  extraEnv](https://zero-to-jupyterhub.readthedocs.io/en/latest/resources/reference.html#singleuser-extraenv)
+  in the configuration reference.
+- imagePullSecrets for all the pods in the Helm chart can now be configured
+  chart wide. See the configuration reference about
+  [imagePullSecret](https://zero-to-jupyterhub.readthedocs.io/en/latest/resources/reference.html#imagepullsecret)
+  and
+  [imagePullSecrets](https://zero-to-jupyterhub.readthedocs.io/en/latest/resources/reference.html#imagepullsecrets)
+  for more details.
+- Deploying the Helm chart in a cluster with a PodSecurityPolicy active is now
+  easier, because the pods' containers now have `securityContext` set on them to
+  run with relatively low permissions which are also configurable if needed.
+- The `autohttps` pod that is running to acquire TLS certificates if
+  `proxy.https.type=letsencrypt` is now more reliably acquiring certificates. If
+  you currently have such issue, do `kubectl delete deploy/autohttps` and
+  `kubectl delete secret proxy-public-tls-acme` and then deploy the Helm chart
+  again with `helm upgrade`.
+
+#### Notable dependencies updated
```

Comment:
this is great - I bet we can auto-generate this in the future (though I suppose we can also just copy/paste this table in the future!)

Reply:
I think so too, but as long as we have a template to copy/paste, most of the work is done. I'm more afraid of the maintaining the automated infrastructure than updating this manually between releases :p